interop/client/client.go

```diff
@@ -270,6 +271,9 @@ func main() {
 	case "unimplemented_service":
 		interop.DoUnimplementedService(testpb.NewUnimplementedServiceClient(conn))
 		grpclog.Infoln("UnimplementedService done")
+	case "pick_first_unary":
+		interop.DoPickFirstUnary(tc, 100)
```

Comment:
small nit: might be better to unparameterize `rpcCount` and hardcode 100 directly into the body of `DoPickFirstUnary` - that, or make have it fully configurable via command arg.

Reply:
I will go with hardcode now. Adding a flag only for one test case also seems bad...